test/controllers/project_skill_controller_test.exs

```diff
@@ -1,14 +1,16 @@
 defmodule CodeCorps.ProjectSkillControllerTest do
-  use CodeCorps.ApiCase
+  use CodeCorps.ApiCase, resource_name: :project_skill
 
   alias CodeCorps.ProjectSkill
   alias CodeCorps.Repo
 
-  defp build_payload, do: %{ "data" => %{"type" => "project-skill", "attributes" => %{}}}
+  @valid_attrs %{}
```

Comment:
actually kind of confused on what this actually does with the Map.merge since its empty?